client/pages/home.jsx

```diff
@@ -3,7 +3,7 @@ import Decks from '../pages/decks';
 
 export default function Home(props) {
   return (
-    <div>
+    <div className='m-1'>
       <Decks />
     </div>
```

Comment:
Instead of this super-small Home component, just use the Decks component directly